vllm/entrypoints/chat_utils.py

```diff
@@ -495,8 +495,9 @@ def _placeholder_str(self, modality: ModalityStr,
             if model_type.startswith("llava"):
                 return self._cached_token_str(self._tokenizer,
                                               hf_config.image_token_index)
-            if model_type in ("chameleon", "deepseek_vl_v2", "internvl_chat",
-                              "skywork_chat", "NVLM_D", "h2ovl_chat"):
+            if model_type in ("ayavision", "chameleon", "deepseek_vl_v2",
```

Comment:
```suggestion
            if model_type in ("aya_vision", "chameleon", "deepseek_vl_v2",
```

The model config has an underscore in the model type